Formula/dynare.rb

```diff
@@ -0,0 +1,85 @@
+class Dynare < Formula
+  desc "Platform for economic models, particularly DSGE and OLG models"
+  homepage "https://www.dynare.org/"
+  url "https://www.dynare.org/release/source/dynare-4.5.3.tar.xz"
+  sha256 "01434f6d3ceaff1891dc771f6a5b39caee787b2ffa1875e5a4e8c673e32ff3d7"
+  head "https://github.com/DynareTeam/dynare.git"
```

Comment:
let's also remove head

Reply:
@ilovezfs is there any way to keep `HEAD`? It's a big reason for us to distribute with Homebrew....